src/warnet/bitcoin.py

```diff
@@ -163,21 +195,26 @@ def get_messages(tank_a: str, tank_b: str, chain: str):
     """
     Fetch messages from the message capture files
     """
+    sclient = get_static_client()
+
     subdir = "" if chain == "main" else f"{chain}/"
     base_dir = f"/root/.bitcoin/{subdir}message_capture"
 
     # Get the IP of node_b
-    cmd = f"kubectl get pod {tank_b} -o jsonpath='{{.status.podIP}}'"
-    tank_b_ip = run_command(cmd).strip()
+    tank_b_pod: V1Pod = sclient.read_namespaced_pod(name=tank_b, namespace=get_default_namespace())
```

Comment:
why not use the get pod helpers in k8s.py?

Reply:
fixed up here: https://github.com/bitcoin-dev-project/warnet/pull/614/commits/6d8ffa091089dfadca4fff2e61331c1e8d109f66
and here: https://github.com/bitcoin-dev-project/warnet/pull/614/commits/e84416bc72411f2dbf00615486428260a5085824